src/jats/__tests__/jats-roundtrip.test.ts

```diff
@@ -164,7 +164,7 @@ describe('JATS transformer', () => {
     expect(parsedOutput.errors).toHaveLength(0)
   })
 
-  test('round-trips JATS XML 2', async () => {
+  test.skip('round-trips JATS XML 2', async () => {
```

Comment:
why are we skipping this test?

Reply:
to be adressed in LEAN-3126

Reply:
The issue has been addressed and resolved under LEAN-3030.